components/bpmn/org.wso2.carbon.bpmn.extensions/src/main/java/org/wso2/carbon/bpmn/extensions/rest/RESTTask.java

```diff
@@ -147,6 +148,7 @@
     private Expression responseHeaderVariable;
     private Expression httpStatusVariable;
     private Expression errorMessageVariable;
+    private Expression defaultVariableSetFlag;
```

Comment:
Shall we rename this to a name that express the functionality
"enableDefaultVariable" ?

Reply:
+1